charts/lily/templates/statefulset.yaml

```diff
@@ -39,37 +39,27 @@ spec:
       imagePullSecrets:
       - name: regcred
       initContainers:
-      - name: initfs
+      - name: init-datastore
         image: {{ required "(root).image.repo expected" .Values.image.repo }}:{{ default (printf "v%s" .Chart.AppVersion) .Values.image.tag }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         command: ["/bin/sh", "-c"]
         args:
           - |
-            mkdir -p /var/lib/lily/keystore
             # generate 6-char random uid to be used in job report names
             if [ ! -f "/var/lib/lily/uid" ]; then
               tr -dc 'a-zA-Z0-9' < /dev/urandom | fold -w ${1:-6} | head -n 1 > /var/lib/lily/uid
             fi
-        volumeMounts:
-        - name: repo-volume
-          mountPath: /var/lib/lily
-        - name: waitjob-script-volume
-          mountPath: /var/lib/lily/waitjob.sh
-          subPath: waitjob.sh
-        resources:
-          requests:
-            cpu: "1000m"
-            memory: "512Mi"
-          limits:
-            cpu: "1000m"
-            memory: "512Mi"
-      {{- if .Values.daemon.importSnapshot.enabled }}
-      - name: chain-import
-        image: {{ required "(root).image.repo expected" .Values.image.repo }}:{{ default (printf "v%s" .Chart.AppVersion) .Values.image.tag }}
-        imagePullPolicy: {{ .Values.image.pullPolicy }}
-        command: ["/bin/sh", "-c"]
-        args:
-          - |{{- include "sentinel-lily.chainImportArgs" . | indent 10 }}
+
+            # create empty keystore
+            if [ ! -f "/var/lib/lily/keystore" ]; then
```

Comment:
Why is the keystore created manually?

Reply:
This was copypasta from the original spike. This was explicitly done (by @iand?) and I did not bother challenging it. I'd prefer not to change it casually for a release, but I can review this later in https://github.com/filecoin-project/sentinel-infra/issues/199 whether it's necessary or not and include that in the comment (or remove).